app/scripts/services/bindingModalUtils.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('BindingModalUtils', [
```

Comment:
Yeah, probably a premature refactor on my part, assuming I would need it for `unbind`. I  will return this to its previous state.